debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/MongoDbSinkConnector.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.common.config.Config;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.Task;
+import org.apache.kafka.connect.sink.SinkConnector;
+
+import io.debezium.annotation.Immutable;
+import io.debezium.config.Configuration;
+
+public class MongoDbSinkConnector extends SinkConnector {
+
+    private static final List<String> REQUIRED_SINK_ACTIONS = Arrays.asList("insert", "update", "remove");
```

Comment:
What's purpose of this attribute?

Reply:
Not used anymore after my previous refactoring. Removed.